src/ansys/pytwin/evaluate/tbrom.py

```diff
@@ -106,14 +106,58 @@ def read_snapshot_size(filepath):
 
     Examples
     --------
-    >>> from pytwin import write_binary
+    >>> from pytwin import read_snapshot_size
     >>> number_data = read_snapshot_size('snapshot.bin')
     """
     with open(filepath, "rb") as f:
         nbdof = struct.unpack("Q", f.read(8))[0]
     return nbdof
 
 
+def snapshot_to_array(snapshot_file, geometry_file):
+    """
+    Create an array containing the x, y, z coordinates and data from geometry
+    and field snapshot files.
+
+    Parameters
+    ----------
+    snapshot_file : str
+        Path of the binary field data file to be read.
+    geometry_file : str
+        Path of the binary points data file to be read.
+
+    Returns
+    -------
+    np.ndarray
+        Return a 2D Numpy array of x,y,z coordinates and snapshot data read.
+        Array has shape (m,n), where:
+
+        - `m = n_g / 3` where `n_g` is the length of the geometry snapshot and.
```

Comment:
I am struggling to understand these formula.
Couldn't we say that:
m is the number of points in the geometry file and n is the dimension of the snapshot field + 3
?